README.md

```diff
@@ -22,6 +22,9 @@ Adds completion of udb command line parameters in bash.
 [**Fancy prompt**](fancy_prompt/README.md)  
 Enhanced prompt support for UndoDB.
 
+[**Follow fork**](follow_fork/README.md)  
+Allows live-record to record the both parent and child after fork() has been called.
```

Comment:
“the both”? :P
Also, backticks around `fork()` please.

Reply:
fixed (embarrassed mumble)